internal/storage/image.go

```diff
@@ -298,8 +316,25 @@ func (svc *imageService) ImageStatus(systemContext *types.SystemContext, nameOrI
 		return nil, err
 	}
 
-	result := svc.buildImageResult(image, cacheItem)
-	return &result, nil
+	return svc.buildImageResult(systemContext, img, cacheItem, ref)
+}
+
+func inspectInfo(sc *types.SystemContext, ref types.ImageReference) (map[string]string, error) {
+	ctx := context.Background()
+	imgSrcRef, err := ref.NewImageSource(ctx, sc)
+	if err != nil {
+		return nil, errors.Wrap(err, "creating image source")
+	}
+	unparsed := image.UnparsedInstance(imgSrcRef, nil)
+	img, err := image.FromUnparsedImage(ctx, sc, unparsed)
+	if err != nil {
+		return nil, errors.Wrap(err, "from unparsed image")
+	}
+	info, err := img.Inspect(ctx)
+	if err != nil {
+		return nil, errors.Wrap(err, "inspecting image")
+	}
+	return info.Labels, nil
```

Comment:
@mrunalp @haircommander we can think about adding some more metadata here, like the OS and ARCH of the image (part of the image status info)

Reply:
Changed it to `Label` that the purpose is more clea.